### ISSUE TITLE
Issue 09 — `Bernoulli` natural-space `getlogpartition` overflows to `Inf` for large logits

### ISSUE THREAD

**Issue by docxology** [ReactiveBayes/ExponentialFamily.jl#298]
# Issue 09 — `Bernoulli` natural-space `getlogpartition` overflows to `Inf` for large logits

**Component:** `src/distributions/bernoulli.jl:92-95`
**Severity:** MEDIUM
**Type:** Numerical / wrong normalization

## Summary
```julia
getlogpartition(::NaturalParametersSpace, ::Type{Bernoulli}) = (η) -> begin
    (η₁,) = unpack_parameters(Bernoulli, η)
    return -log(logistic(-η₁))
end
```
For η₁ ≳ 709, `logistic(-η₁) → 0` in Float64, so `-log(0) = Inf`, whereas the true
`A(η) = log1pexp(η) ≈ η` is finite.

## Evidence / repro (Julia 1.12.6)
```julia
getlogpartition(NaturalParametersSpace(), Bernoulli)([1000.0])  # => Inf (true ≈ 1000.0)
```

## Impact
The log-partition (normalizer / `compute_logscale`) is `Inf` for a very confident
`Bernoulli`, corrupting normalization while the (stable) mean gradient stays finite.

## Proposed fix
```julia
return log1pexp(η₁)     # StatsFuns.log1pexp
```

**Comment by bvdmitri** [comment]
Fixed in #301 and released in v2.6.0 (see the `[2.6.0]` CHANGELOG entry): `Bernoulli` natural-space `getlogpartition` now uses `log1pexp`, so large logits no longer overflow to `Inf` (`src/distributions/bernoulli.jl`). Regression test added in 12ccde0.
